app/controllers/serial_searches_controller.rb

```diff
@@ -8,9 +8,16 @@ def index
       @title = 'Serial Searches'
       per_page = params[:per_page] || 50
       page = params[:page] || 1
+
       @serial_searches_count = SerialSearch.count
       @ip_addresses_count = IpAddress.count
-      @serial_searches = SerialSearch.order(sort_column + ' ' + sort_direction).page(page).per(per_page)
+
+      if params[:query].present?
+        @serial_searches = SerialSearch.text_search(params[:query]).page(page).per(per_page)
```

Comment:
Perhaps this is the cause for the confision..Is this what you're referring to? 
http://stackoverflow.com/questions/18198963/with-rails-4-model-scoped-is-deprecated-but-model-all-cant-replace-it